qiita_db/archive.py

```diff
@@ -149,6 +149,8 @@ def retrieve_feature_values(cls, archive_merging_scheme=None,
         ----------
         archive_merging_scheme : optional, str
             The name of the archive_merging_scheme to retrieve
+        features : list of str
```

Comment:
Minor but can you add `features : list of str, optional`.

Reply:
yup